server/src/test/java/io/crate/execution/engine/sort/SortTest.java

```diff
@@ -70,4 +73,24 @@ public void testParallelSortYieldsTheSameResultAsListSort() throws Exception {
                 executor
             ).get(5, TimeUnit.SECONDS));
     }
+
+    @Test
+    public void test_parallel_sort_accounts_for_list_shallow_size() throws Exception {
+        int length = 500;
+        int minItemsPerThread = 100;
+        int numAvailableThreads = 5; // Imitating parallel threads only to hit specific code path.
+        PlainRamAccounting accounting = new PlainRamAccounting();
+        AccountableList<Integer> numbers = new AccountableList<>(accounting::addBytes);
+        for (int i = 0; i < length; i++) {
+            numbers.add(randomInt());
+        }
+        Sort.parallelSort(
+                numbers,
+                Comparator.comparingInt(x -> x),
+                minItemsPerThread,
+                numAvailableThreads,
+                executor
+         ).get(5, TimeUnit.SECONDS);
+        assertThat(accounting.totalBytes()).isEqualTo(2252L);
```

Comment:
sorting correctness is already testsed in `testParallelSortYieldsTheSameResultAsListSort`. 

Here we only check accounting in parallel code path. None-parallel execution is checked in OffsetValueFunctionsTest (see test update)

Reply:
Wouldn't it be better to create a dedicated test to validate the behaviour of the new `AccountableList` standalone?